next/src/global/types.ts

```diff
@@ -0,0 +1 @@
+type PlatformType = 'instagram' | 'youtube' | 'tiktok';
```

Comment:
I do not know if it's a good idea to declare `PlatformType` for global types. It is more like component specific types.

Reply:
It will be used in at least two sections where there will be a SocialLinks, that's also why I've got a SocialLink as a global UI component.